packages/deployment/app/cumulus_api.template.yml

```diff
@@ -180,21 +166,28 @@ Resources:
           stackName: {{../stackName}}
           public_buckets: {{{collectBuckets ../parent.buckets "public"}}}
           protected_buckets: {{{collectBuckets ../parent.buckets "protected"}}}
-      {{# if this.urs}}
+      {{# ifEquals this.urs_redirect "token"}}
```

Comment:
What about existing users with `urs: true` on their custom lambdas? This would break them

Reply:
Actually, I just remember that this change (or something like it) is necessary because otherwise the distribution API stack will need a reference to the `backendRestApi` to build the `TOKEN_REDIRECT` env var. 

Since backendRestApi depends on distributionRestApi, if we made distributionRestApi depend on backendRestApi, we'd have a circular dependency.